### PR TITLE
Update commands.md

### PR DESCRIPTION
**Pull request by thezackm**
update to add notes re: supported "shell" options

### CODE REVIEW

File: docs/apis/commands.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ The following table describes the properties of the `commands` API. The API acce
 | Name | Type | Default | Description |
 |---:|:---:|:---:|---|
 | `run` | string |  | Command or application that you want to run. It accepts any valid shell command. You can also use environment variables with the format `$$ENV_VAR_NAME` |
-| `shell` | string | `/bin/sh` (Linux) `cmd` (Windows) | Shell to use when executing the command defined in `run` |
+| `shell` | string | `/bin/sh` (Linux) `cmd` (Windows) | Shell to use when executing the command defined in `run`. All native Linux Shells, Windows CMD, and Windows PowerShell v1-5.x (`powershell`) and v6+ (`pwsh`) are supported. |
```
Comment (theletterf) [inline docs/apis/commands.md]: s/Shells/shells

Comment (thezackm) [reply docs/apis/commands.md]: resolved with commit 06bd27e

**Comment by thezackm** [comment]
@theletterf @ardias - am I good to merge this one?

**Comment by theletterf** [comment]
Yep!

**Pull request merged**